tests/integration/test_fetch_part_from_auxiliary_zookeeper/test.py

```diff
@@ -0,0 +1,40 @@
+
+
+import pytest
+from helpers.client import QueryRuntimeException
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance("node", main_configs=["configs/zookeeper_config.xml"], with_zookeeper=True)
+
+
+@pytest.fixture(scope="module")
+def start_cluster():
+    try:
+        cluster.start()
+
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+
+def test_fetch_part_from_allowed_zookeeper(start_cluster):
```

Comment:
Looks like copy-paste from `test_fetch_partition_from_auxiliary_zookeeper/test.py`. Do you consider using `@pytest.mark.parametrize`?

Reply:
which is different， i just chang test_fetch_partition_from_auxiliary_zookeeper/test.py function name test_fetch_part_from_auxiliary_zookeeper  to test_fetch_partition_from_auxiliary_zookeeper


test_fetch_part_from_auxiliary_zookeeper/test.py, which is a new test for this feature

Reply:
But tests are almost the same, the only difference is usage of `ATTACH PART` instead of `ATTACH PARTITION` and these features are similar, so let's use `@pytest.mark.parametrize` in `test_fetch_partition_from_auxiliary_zookeeper/test.py::test_fetch_partition_from_allowed_zookeeper` and remove `test_fetch_part_from_auxiliary_zookeeper/test.py::test_fetch_part_from_allowed_zookeeper`

Reply:
3ks,done